### PR TITLE
Advanced Selectors lesson: Provide a link to documentation on CSS combinators

### PR DESCRIPTION
**Pull request by DevVivan**
## Because

I have provided a link for MDN docs on the CSS combinators similar to the other sections in this lesson. Students can check it out to find out some extra information on this topic and also see some more examples of child and sibling combinators in action.

## This PR

- A docs link to information & examples on CSS combinators.

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: intermediate_html_css/intermediate_css_concepts/advanced_selectors.md [hunk 1]
```diff
@@ -86,6 +86,8 @@ Finally, if we want to select all of an element's siblings and not just the firs
 
 Just like the descendant combinator, these selectors don't have any special specificity rules - their specificity score will just be made up of their component parts.
 
+You can also check out [the docs](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Combinators) on this to find out more about combinators.
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/advanced_selectors.md:89]:
```suggestion
This [MDN article on combinators](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Combinators) provides a good overview if you want to learn more about them.
```

I edited the sentence a bit to make the link more descriptive. What do you think of this wording?

**Comment by ManonLef** [comment]
I think this is a nice addition that is in line with other sections of this lesson where there's a link to the MDN docs.

File: intermediate_html_css/intermediate_css_concepts/advanced_selectors.md (hunk 1 above)
Comment (DevVivan) [reply intermediate_html_css/intermediate_css_concepts/advanced_selectors.md:89]: Yep! That works very well. I'll commit this.

**Comment by DevVivan** [comment]
Commited @ManonLef :)

**Comment by ManonLef** [comment]
Thanks again @DevVivan 💪

**Pull request merged**